wayland-client/src/lib.rs

```diff
@@ -153,7 +153,7 @@ mod anonymous_object {
     ///
     /// A special Interface implementation representing an
     /// handle to an object for which the interface is not known.
-    #[derive(Clone, Eq, PartialEq)]
+    #[derive(Debug, Clone, Eq, PartialEq)]
     pub struct AnonymousObject(pub(crate) Proxy<AnonymousObject>);
```

Comment:
I'd rather this one have a similar `Debug` impl as the objects generated by `wayland-scanner`.